### PR TITLE
Fix for #8788 JabRef not showing contents of shared database library

### PR DESCRIPTION
**Pull request by falschgeldkind**
Fixes: https://github.com/JabRef/jabref/issues/8788

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/logic/shared/DBMSProcessor.java [hunk 1]
```diff
@@ -26,6 +16,7 @@
 import org.jabref.model.entry.field.FieldFactory;
 import org.jabref.model.entry.types.EntryTypeFactory;
 
+import org.jbibtex.BibTeXDatabase;
```
[the remark this answers is not in the document]
Comment (falschgeldkind) [reply src/main/java/org/jabref/logic/shared/DBMSProcessor.java:19]: yeah.. I don't really know why the IDE did that. I can't remember doing that on myself. feel free to change anything

[the remark this answers is not in the document]
Comment (Siedlerchr) [reply src/main/java/org/jabref/logic/shared/DBMSProcessor.java:19]: You can import JabRefs' codestyle, otherwise I will take a look at it later.

**Comment by Siedlerchr** [comment]
@falschgeldkind 
I fixed the check style issues and used Google Guava's LIsts.partition() method.
I cannot push to your repo, so can you please add a changelog entry to the Changelog.md?  

PS: For the next time this is helpful (and better create a branch next time) : https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by koppor** [comment]
I merge as this fixes the issue.

Nevertheless, @falschgeldkind Could you comment about the "magic number" 500? The latency between the desktop app and the server was too high and thus the connection went into a time out?

**Comment by Siedlerchr** [comment]
The number of parameters in the in clause was too high. Limit of postgres was hit. See issue for detail

**Pull request merged**